src/closures/buoyancy_gradients.jl

```diff
@@ -13,64 +13,87 @@ function buoyancy_gradients(
     param_set::APS,
     bg_model::EnvBuoyGrad{FT, EBG},
 ) where {FT <: Real, EBG <: AbstractEnvBuoyGradClosure}
-
     g = TCP.grav(param_set)
-    molmass_ratio = TCP.molmass_ratio(param_set)
     R_d = TCP.R_d(param_set)
     R_v = TCP.R_v(param_set)
+    T_0 = TCP.T_0(param_set)
+    L_v0 = TCP.LH_v0(param_set)
+    L_s0 = TCP.LH_s0(param_set)
+
+    if bg_model.en_cld_frac < FT(1)
+        T = bg_model.T_unsat
+        Tv_gm = bg_model.Tv_gm
+        ts_unsat = TD.PhaseEquil_pTq(param_set, bg_model.p, T, bg_model.qt_unsat)
+        R_m = TD.gas_constant_air(param_set, ts_unsat)
+        cp_m = TD.cp_m(param_set, ts_unsat)
 
-    phase_part = TD.PhasePartition(FT(0), FT(0), FT(0)) # assuming R_d = R_m
-    Π = TD.exner_given_pressure(param_set, bg_model.p, phase_part)
+        ∂b∂M_unsat = (g / Tv_gm) * (R_m / R_d / cp_m)
+        ∂b∂qt_unsat = (g / Tv_gm) * (R_m / cp_m * (R_d * T_0 - L_v0) + T / R_d * (R_v - R_d))
```

Comment:
```suggestion
        ∂b∂qt_unsat = (g / Tv_gm / R_d) * (R_m / cp_m * (R_d * T_0 - L_v0) + T * (R_v - R_d))
```
1/R_d factor is missing in first term.